src/runtime_src/core/include/xrt/detail/pimpl.h

```diff
@@ -28,12 +28,12 @@ namespace xrt { namespace detail {
 template<typename ImplType>
 class pimpl
 {
+protected:
+  ~pimpl() = default;
```

Comment:
Do you need it at all?

Reply:
Likely not.  But at least protected now, not that it matters really.